datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -234,6 +253,16 @@ select column1, nth_value(column3, 2 order by column2, column4 desc) from array_
 b [4, 5, 6]
 w [9, 5, 2]
 
+query ?
```

Comment:
Can you please add a test for 

```
-- default ordering (and show that desc is respected)
select array_agg(DISTINCT column2 order by column2) from array_agg_order_list_table;
```

Also a query with a `GROUP BY` as it goes through a different code path

Also the negative case (no order by but with distinct) -- to show the error message is wired up correctly

Reply:
yes exactly